crates/bevy_a11y/src/lib.rs

```diff
@@ -97,11 +96,6 @@ impl From<Node> for AccessibilityNode {
     }
 }
 
-/// Resource representing which entity has keyboard focus, if any.
-#[derive(Resource, Default, Deref, DerefMut, Reflect)]
```

Comment:
I didn't keep the Deref / DerefMut derives around: do you want me to readd them?